pkg/trace/obfuscate/sql_tokenizer.go

```diff
@@ -68,18 +68,25 @@ const (
 	FilteredBracketedIdentifier
 )
 
+const escapeCharacter = '\\'
+
 // SQLTokenizer is the struct used to generate SQL
 // tokens for the parser.
 type SQLTokenizer struct {
-	rd       *strings.Reader // the "rune" reader
-	pos      int             // byte offset of lastChar
-	lastChar rune            // last read rune
-	err      error           // any error occurred while reading
+	rd            *strings.Reader // the "rune" reader
+	pos           int             // byte offset of lastChar
+	lastChar      rune            // last read rune
+	err           error           // any error occurred while reading
+	ignoreEscape  bool            // indicates we should not try to escape quotes
```

Comment:
Maybe `literalEscape` is indeed better than `ignoreEscape`, since we aren't really ignoring it. Also, please add an empty line above this one. I feel that these two belong together in another group, as settings, while the others are strictly related to parsing.